src/app.jsx

```diff
@@ -367,6 +367,7 @@ class Application extends React.Component {
         case 'sync':
         case 'unmount':
         case 'unpause':
+        case 'rename': // rename event is available starting podman v4.1; until then the container does not get refreshed after renaming
             this.updateContainerAfterEvent(event.Actor.ID, system);
```

Comment:
I've tested this on an older podman and it's a bit annoying that there is no visual feedback from the rename action. I think we should do something about that before merging the PR.

When /rename returns 204 did the rename already happen? If so we can manually refresh the container status I suppose until everything is podman 3.4

Reply:
I agree and I would refresh it manually only if someone helped me how to call Application instance method correctly from handler code and not breaking architecture? I am new to react.

Reply:
I figured out how to refresh the status of renamed container manually, it should be fixed now.